src/index.ts

```diff
@@ -10,14 +10,27 @@ async function run() {
 
     const buildParameters = await BuildParameters.create();
     const baseImage = new ImageTag(buildParameters);
-    if (buildParameters.kubeConfig) {
-      core.info('Building with Kubernetes');
-      await Kubernetes.runBuildJob(buildParameters, baseImage);
-    } else {
-      // Build docker image
-      // TODO: No image required (instead use a version published to dockerhub for the action, supply credentials for github cloning)
-      const builtImage = await Docker.build({ path: actionFolder, dockerfile, baseImage });
-      await Docker.run(builtImage, { workspace, ...buildParameters });
+    let builtImage;
+
+    switch (buildParameters.remoteBuildCluster) {
+      case 'k8s':
+        core.info('Building with Kubernetes');
+        await Kubernetes.runBuildJob(buildParameters, baseImage);
+        break;
+
+      case 'aws':
+        core.info('Building with AWS');
+        await AWS.runBuildJob(buildParameters, baseImage);
+        break;
+
+      // default and local case
+      default:
+        core.info('Building locally');
+        // Build docker image
+        // TODO: No image required (instead use a version published to dockerhub for the action, supply credentials for github cloning)
```

Comment:
Do these comments belong to case 'aws'? Or how should we read this?

Reply:
Cleaned up, was an old message now